source/nodejs/adaptivecards/src/adaptive-applet.ts

```diff
@@ -714,6 +748,52 @@ export class AdaptiveApplet {
         }
     }
 
+    private async internalSendDataQueryRequestAsync(request: ActivityRequest) {
+        if (!this.channelAdapter) {
+            throw new Error("internalSendDataQueryRequestAsync: channel adapter not set");
+        }
+        this._choiceSet?.showLoadingIndicator();
+        let response = undefined;
+        try {
+            response = await this.channelAdapter.sendRequestAsync(request);
+        } catch (error) {
+            logEvent(Enums.LogLevel.Error, "Activity request failed: " + error);
+        }
+        this._choiceSet?.removeLoadingIndicator();
+        if (response) {
+            if (response instanceof SuccessResponse) {
```

Comment:
can these two if blocks be combined into one?

Reply:
Different blocks are inside.